src/app/zero-shot-details/component/zero-shot-details.component.ts

```diff
@@ -110,6 +116,15 @@ export class ZeroShotDetailsComponent
     this.subscriptions$.push(project$.subscribe((project) => this.project = project));
     combineLatest(tasks$).subscribe(() => this.prepareInformationSource(projectId));
 
+    [this.downloadedModelsQuery$, this.downloadedModelsList$] = this.informationSourceApolloService.getModelProviderInfo();
+    this.subscriptions$.push(
+      this.downloadedModelsList$.subscribe((downloadedModels) => {
+        this.downloadedModels = downloadedModels;
+        this.createModelsDownloadedStateList();
+      }));
+
+    this.isManaged = ConfigManager.getIsManaged();
```

Comment:
see above (config manager is requested before the init is done)

Reply:
to prevent this issue in the future I added an error throw to the config manager if the value is requested before it's initialized ([commit link](https://github.com/code-kern-ai/refinery-ui/pull/44/commits/cb1d7678c49fdbb64c37df4d205210c22fa5105a))

Reply:
still has a bug, fixing it atm

Reply:
check is now included (hopfully :D ) - its a console log now

Reply:
Also the check if init is added so I am resolving the conversation :)